src/lib/hooks/useNotificationHandler.ts

```diff
@@ -60,6 +61,28 @@ export function useNotificationsHandler() {
   // Safety to prevent double handling of the same notification
   const prevDate = React.useRef(0)
 
+  React.useEffect(() => {
+    if (!isAndroid) return
+
+    Notifications.setNotificationChannelAsync('chat-messages', {
+      name: 'Chat',
+      importance: Notifications.AndroidImportance.MAX,
+      sound: 'dm.mp3',
+      showBadge: true,
+      vibrationPattern: [250],
+      lockscreenVisibility: Notifications.AndroidNotificationVisibility.PRIVATE,
+    })
+
+    Notifications.setNotificationChannelAsync('chat-messages-muted', {
+      name: 'Chat - Muted',
+      importance: Notifications.AndroidImportance.MAX,
+      sound: null,
+      showBadge: true,
+      vibrationPattern: [250],
+      lockscreenVisibility: Notifications.AndroidNotificationVisibility.PRIVATE,
+    })
+  }, [])
```

Comment:
Creating two new notification channels for the custom sounds. We'll have to create others for stuff like "Likes" and "Replies", etc. if we want to have more control over those in the future (regarding sound playback).

The user also gets some good configuration on Android through the different channels so we _should_ create separate ones for different notification reasons. Right now they all get grouped into one bucket.